python/simulation/fake_raft.py

```diff
@@ -100,7 +96,6 @@ def get_file_suffix(filepath):
 
 def get_template_files(root_folder, sensor_type, sensor_id, process_name, **kwargs):
     """ Get files that serve as templates for a particular process_type
-
```

Comment:
numpy docstring examples seem to have the whitespace before things like the Parameters, Returns, etc sections.  I think it enhances readability to retain them.